hazelcast/src/main/java/com/hazelcast/nio/tcp/handlermigration/StaticMigrationStrategy.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.nio.tcp.handlermigration;
+
+import com.hazelcast.nio.tcp.MigratableHandler;
+import java.util.Set;
+
+/**
+ * Simple Migration Strategy for Handlers.
+ *
+ * It attempts to trigger a migration if a ratio between least busy and most busy selectors
+ * exceeds {@link #MIN_MAX_RATIO_MIGRATION_THRESHOLD}.
+ *
+ * Once a migration is triggered it tries to find the busiest handler registered in
+ * {@link LoadImbalance#sourceSelector} which wouldn't cause
+ * overload of the {@link LoadImbalance#destinationSelector} after a migration.
+ *
+ */
+class StaticMigrationStrategy implements MigrationStrategy {
```

Comment:
I would not call this 'static' because it is everything but static. Perhaps just call it DefaultMigrationStrategy.